models/gc_stoic_baseline/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM mhubai/base:latest
+
+# Update authors label
+LABEL authors="sil.vandeleemput@radboudumc.nl"
+
+# Install PyTorch 2.0.1 (CUDA enabled)
+RUN pip3 install --no-cache-dir torch==2.0.1+cu118 -f https://download.pytorch.org/whl/torch_stable.html
+
+# Install stoic2021 baseline algorithm
+#  - Git clone to tmp directory (main branch, commit 88611ed1d49145db547f3db40b6fa917a403c239)
+#  - Extract relevant files to src
+#  - Remove tmp directory
+RUN git clone https://github.com/luukboulogne/stoic2021-baseline.git /tmp/algorithm && \
+    cd /tmp/algorithm && git reset --hard 88611ed1d49145db547f3db40b6fa917a403c239 && cd /app && \
+    mkdir src && \
+    mv /tmp/algorithm/process.py src/process.py && \
+    mv /tmp/algorithm/utils.py src/utils.py && \
+    mv /tmp/algorithm/algorithm src/algorithm && \
+    mv /tmp/algorithm/requirements.txt src/requirements.txt && \
+    rm -rf /tmp/algorithm
+
+# Install stoic2021 dependencies
+RUN pip3 install --no-cache-dir -r src/requirements.txt
+
+# FIXME: temporary fix waiting for the latest base image update
+# Clone the main branch of MHubAI/models
+RUN git stash \
```

Comment:
This code is only functional upon approval and integration in the mhubai/models repository. 
To avoid this and to allow for testing without modifications, please use the recently introduced model import build utility instead:

```Dockerfile
# Import the MHub model definiton
ARG MHUB_MODELS_REPO
RUN buildutils/import_mhub_model.sh gc_stoic_baseline ${MHUB_MODELS_REPO}
```

Reply:
Done